target/linux/ramips/image/rt305x.mk

```diff
@@ -1190,7 +1190,7 @@ define Device/zte_mf283plus
   DEVICE_VENDOR := ZTE
   DEVICE_MODEL := MF283+
   DEVICE_PACKAGES := kmod-usb-ohci kmod-usb2 kmod-usb-net-qmi-wwan uqmi \
-	kmod-usb-serial kmod-usb-serial-option
+	kmod-usb-serial kmod-usb-serial-option kmod-leds-gpio
```

Comment:
kmod-leds-gpio is selected in ramips/Makefile

Reply:
Then this needs further investigation, without this change LEDs wouldn't work after I fixed booting. I will do some testing today in the evening/

Reply:
It's maybe broken if you _not_ choose per-device rootfs ...

Reply:
I grepped kmod-leds-gpio in ramips and there were only three occurrences in rt305x subtarget in total.

Reply:
I guess I must have had something messed up in my configuration then, or it must have been not "from scratch". 
After I rebased to recent master I did 3 builds with fresh configurations:
- single device
- multiple devices with single profile selected for DUT - without per-device rootfs
- multiple devices with single profile selected for DUT - with per-device rootfs,
And all of them worked, so I decided to drop this patch. LZMA-loader fix is still required, though.